pandas/core/tools/datetimes.py

```diff
@@ -340,6 +340,17 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
 
     tz = 'utc' if utc else None
 
+    def _maybe_convert_to_utc(arg, utc):
+        if utc:
+            if isinstance(arg, ABCSeries):
+                arg = arg.dt.tz_localize('UTC')
+            elif isinstance(arg, DatetimeIndex):
+                if arg.tz is None:
+                    arg = arg.tz_localize('UTC')
+                else:
+                    arg = arg.tz_convert('UTC')
```

Comment:
this is a BIG change here. we *don't* convert existing tz-aware timezones. why are you doing this?